packages/types/src/documents/app/row.ts

```diff
@@ -32,3 +32,13 @@ export interface Row extends Document {
   tableId?: string
   [key: string]: any
 }
+
+export interface DeleteRows {
+  rows: (Row | string)[]
+}
+
+export interface DeleteRow {
+  _id: string
+}
+
+export type DeleteRowRequest = DeleteRows | DeleteRow
```

Comment:
These should be moved to the `src/api/web` package to indicate that it's a web request